{"uid":"b2078bc0f1de3a21","name":"test_delete_user_authorized_as_another_user","fullName":"tests.test_user_delete.TestUserDelete#test_delete_user_authorized_as_another_user","historyId":"f921dd3d5d139c34912ef23ea50982dc","time":{"start":1710997643989,"stop":1710997648577,"duration":4588},"description":"This test verifies that one user cannot delete another user","descriptionHtml":"<p>This test verifies that one user cannot delete another user</p>\n","status":"passed","statusMessage":"XPASS This test is expected to fail due to a known defect","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":0,"retriesStatusChange":false,"beforeStages":[],"testStage":{"description":"This test verifies that one user cannot delete another user","status":"passed","statusMessage":"XPASS This test is expected to fail due to a known defect","steps":[{"name":"POST request to URL '/user/'","time":{"start":1710997643989,"stop":1710997644910,"duration":921},"status":"passed","steps":[],"attachments":[],"parameters":[],"hasContent":false,"attachmentStep":false,"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0},{"name":"POST request to URL '/user/'","time":{"start":1710997644910,"stop":1710997645827,"duration":917},"status":"passed","steps":[],"attachments":[],"parameters":[],"hasContent":false,"attachmentStep":false,"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0},{"name":"POST request to URL '/user/login'","time":{"start":1710997645828,"stop":1710997646241,"duration":413},"status":"passed","steps":[],"attachments":[],"parameters":[],"hasContent":false,"attachmentStep":false,"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0},{"name":"DELETE request to URL '/user/94875'","time":{"start":1710997646242,"stop":1710997647517,"duration":1275},"status":"passed","steps":[],"attachments":[],"parameters":[],"hasContent":false,"attachmentStep":false,"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0},{"name":"GET request to URL '/user/94875'","time":{"start":1710997647517,"stop":1710997648573,"duration":1056},"status":"passed","steps":[],"attachments":[],"parameters":[],"hasContent":false,"attachmentStep":false,"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0}],"attachments":[],"parameters":[],"hasContent":true,"attachmentStep":false,"shouldDisplayMessage":true,"stepsCount":5,"attachmentsCount":0},"afterStages":[],"labels":[{"name":"epic","value":"User Deletion Tests"},{"name":"tag","value":"@pytest.mark.xfail(reason='This test is expected to fail due to a known defect')"},{"name":"parentSuite","value":"tests"},{"name":"suite","value":"test_user_delete"},{"name":"subSuite","value":"TestUserDelete"},{"name":"host","value":"fv-az1498-325"},{"name":"thread","value":"2279-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"tests.test_user_delete"},{"name":"resultFormat","value":"allure2"}],"parameters":[],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","retries":[],"categories":[],"history":{"statistic":{"failed":0,"broken":0,"skipped":151,"passed":14,"unknown":0,"total":165},"items":[{"uid":"1f3da1d68b4a5b66","reportUrl":"https://Badx86.github.io/LearnQA_Python_API/166//#testresult/1f3da1d68b4a5b66","status":"passed","statusDetails":"XPASS This test is expected to fail due to a known defect","time":{"start":1710911206434,"stop":1710911211227,"duration":4793}},{"uid":"1a8b25e6cd9d97da","reportUrl":"https://Badx86.github.io/LearnQA_Python_API/165//#testresult/1a8b25e6cd9d97da","status":"passed","statusDetails":"XPASS This test is expected to fail due to a known defect","time":{"start":1710824823388,"stop":1710824827916,"duration":4528}},{"uid":"ea104e41ac6888f","reportUrl":"https://Badx86.github.io/LearnQA_Python_API/164//#testresult/ea104e41ac6888f","status":"passed","statusDetails":"XPASS This test is expected to fail due to a known defect","time":{"start":1710738529439,"stop":1710738533918,"duration":4479}},{"uid":"ae4029e4fc100dec","reportUrl":"https://Badx86.github.io/LearnQA_Python_API/163//#testresult/ae4029e4fc100dec","status":"passed","statusDetails":"XPASS This test is expected to fail due to a known defect","time":{"start":1710652102279,"stop":1710652106835,"duration":4556}},{"uid":"e0b196045d52a1ba","reportUrl":"https://Badx86.github.io/LearnQA_Python_API/162//#testresult/e0b196045d52a1ba","status":"passed","statusDetails":"XPASS This test is expected to fail due to a known defect","time":{"start":1710565613277,"stop":1710565617899,"duration":4622}},{"uid":"108534a550529a9d","reportUrl":"https://Badx86.github.io/LearnQA_Python_API/161//#testresult/108534a550529a9d","status":"passed","statusDetails":"XPASS This test is expected to fail due to a known defect","time":{"start":1710479299779,"stop":1710479304944,"duration":5165}},{"uid":"6cbfb2e0cc3b8a68","reportUrl":"https://Badx86.github.io/LearnQA_Python_API/160//#testresult/6cbfb2e0cc3b8a68","status":"passed","statusDetails":"XPASS This test is expected to fail due to a known defect","time":{"start":1710392834357,"stop":1710392838882,"duration":4525}},{"uid":"adb729035fed4f83","reportUrl":"https://Badx86.github.io/LearnQA_Python_API/159//#testresult/adb729035fed4f83","status":"passed","statusDetails":"XPASS This test is expected to fail due to a known defect","time":{"start":1710306464427,"stop":1710306469271,"duration":4844}},{"uid":"4bdfdb63cf0428e1","reportUrl":"https://Badx86.github.io/LearnQA_Python_API/158//#testresult/4bdfdb63cf0428e1","status":"passed","statusDetails":"XPASS This test is expected to fail due to a known defect","time":{"start":1710220136398,"stop":1710220141411,"duration":5013}},{"uid":"bc9da528b9636a45","reportUrl":"https://Badx86.github.io/LearnQA_Python_API/157//#testresult/bc9da528b9636a45","status":"passed","statusDetails":"XPASS This test is expected to fail due to a known defect","time":{"start":1710133645361,"stop":1710133649983,"duration":4622}},{"uid":"8da9cd970df57af8","reportUrl":"https://Badx86.github.io/LearnQA_Python_API/156//#testresult/8da9cd970df57af8","status":"passed","statusDetails":"XPASS This test is expected to fail due to a known defect","time":{"start":1710047174971,"stop":1710047180001,"duration":5030}},{"uid":"b2161edf1de21ec8","reportUrl":"https://Badx86.github.io/LearnQA_Python_API/155//#testresult/b2161edf1de21ec8","status":"passed","statusDetails":"XPASS This test is expected to fail due to a known defect","time":{"start":1709960883291,"stop":1709960888201,"duration":4910}},{"uid":"4a8f3cacd5cda1da","reportUrl":"https://Badx86.github.io/LearnQA_Python_API/154//#testresult/4a8f3cacd5cda1da","status":"passed","statusDetails":"XPASS This test is expected to fail due to a known defect","time":{"start":1709874472893,"stop":1709874477504,"duration":4611}},{"uid":"580b717d8d7cf28","reportUrl":"https://Badx86.github.io/LearnQA_Python_API/153//#testresult/580b717d8d7cf28","status":"skipped","statusDetails":"XFAIL This test is expected to fail due to a known defect\n\nAssertionError: Unexpected status code 200\nassert 200 == 400\n +  where 200 = <Response [200]>.status_code","time":{"start":1709788001418,"stop":1709788005185,"duration":3767}},{"uid":"286da7f526e96654","reportUrl":"https://Badx86.github.io/LearnQA_Python_API/152//#testresult/286da7f526e96654","status":"skipped","statusDetails":"XFAIL This test is expected to fail due to a known defect\n\nAssertionError: Unexpected status code 200\nassert 200 == 400\n +  where 200 = <Response [200]>.status_code","time":{"start":1709701615062,"stop":1709701619021,"duration":3959}},{"uid":"3b793e5fb9920855","reportUrl":"https://Badx86.github.io/LearnQA_Python_API/151//#testresult/3b793e5fb9920855","status":"skipped","statusDetails":"XFAIL This test is expected to fail due to a known defect\n\nAssertionError: Unexpected status code 200\nassert 200 == 400\n +  where 200 = <Response [200]>.status_code","time":{"start":1709615240592,"stop":1709615244448,"duration":3856}},{"uid":"2080d8a043a94bbf","reportUrl":"https://Badx86.github.io/LearnQA_Python_API/150//#testresult/2080d8a043a94bbf","status":"skipped","statusDetails":"XFAIL This test is expected to fail due to a known defect\n\nAssertionError: Unexpected status code 200\nassert 200 == 400\n +  where 200 = <Response [200]>.status_code","time":{"start":1709530884615,"stop":1709530888095,"duration":3480}},{"uid":"9246ba2951849e5d","reportUrl":"https://Badx86.github.io/LearnQA_Python_API/149//#testresult/9246ba2951849e5d","status":"skipped","statusDetails":"XFAIL This test is expected to fail due to a known defect\n\nAssertionError: Unexpected status code 200\nassert 200 == 400\n +  where 200 = <Response [200]>.status_code","time":{"start":1709442428244,"stop":1709442432166,"duration":3922}},{"uid":"b75b37e511f98dd9","reportUrl":"https://Badx86.github.io/LearnQA_Python_API/148//#testresult/b75b37e511f98dd9","status":"skipped","statusDetails":"XFAIL This test is expected to fail due to a known defect\n\nAssertionError: Unexpected status code 200\nassert 200 == 400\n +  where 200 = <Response [200]>.status_code","time":{"start":1709356053858,"stop":1709356057998,"duration":4140}},{"uid":"f788fc51e3364fec","reportUrl":"https://Badx86.github.io/LearnQA_Python_API/147//#testresult/f788fc51e3364fec","status":"skipped","statusDetails":"XFAIL This test is expected to fail due to a known defect\n\nAssertionError: Unexpected status code 200\nassert 200 == 400\n +  where 200 = <Response [200]>.status_code","time":{"start":1709269671408,"stop":1709269675197,"duration":3789}}]},"tags":["@pytest.mark.xfail(reason='This test is expected to fail due to a known defect')"]},"source":"b2078bc0f1de3a21.json","parameterValues":[]}